### PR TITLE
[Linux] Add binary name support during app search

### PR DESCRIPTION
**Pull request by nazar-pc**
It works, while not perfect yet.
Ideally there should be support for search priorities. When match is found in app name - it should be higher that when it is found in binary name.

Technically implementation is similar to Windows.

### CODE REVIEW

File: app/main/plugins/core/basic-apps/linux.js [hunk 1]
```diff
@@ -103,7 +103,10 @@ const findIcon = (icon) => {
   )).find(fs.existsSync)
 }
 
-export const toString = ({ name }) => `${name} ${getAbbr(name)}`
+export const toString = ({ name, exec }) => {
+  const binaryName = exec.split('/').pop().split(' ').shift()
```
Comment (KELiON) [inline app/main/plugins/core/basic-apps/linux.js]: Is it possible to have `exec === undefined` here? It will raise an exception

Comment (nazar-pc) [inline app/main/plugins/core/basic-apps/linux.js]: According to https://github.com/nazar-pc/cerebro/blob/ae0d8a6d81b09c7723107fb74551e8c8b2e6376e/app/main/plugins/core/basic-apps/linux.js#L76-L76 it shouldn't be the case, should always be a string.

Comment (KELiON) [inline app/main/plugins/core/basic-apps/linux.js]: @nazar-pc yep, I see! So, just fix eslint and iti s ready to merge!

Comment (nazar-pc) [inline app/main/plugins/core/basic-apps/linux.js]: Should be good now

**Pull request merged**